projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -38,4 +39,38 @@ export function updateVDKMenu(commands: CommandRegistry) {
             }).catch((e) => console.log(e));
         },
     });
+
+    commands.addCommand("jp-vdk:menu-delete", {
+        label: "Delete",
+        caption: 'Execute VDK Delete Command',
+        execute: async () => {
+            let defaultJobName = sessionStorage.getItem("current-path")!.substring(sessionStorage.getItem("current-path")!.lastIndexOf("/"));
+            showDialog({
+                title: "Delete Job",
+                body: <DeleteJobDialog jobName={defaultJobName}></DeleteJobDialog>,
+                buttons: [Dialog.okButton(), Dialog.cancelButton()],
```

Comment:
nit: maybe print the team as well (since could be dynamically changed, so it is currently shadowed for the user)